run.py

```diff
@@ -158,7 +172,7 @@ async def gateway_player(self, data: Player):
         ctx = self.tokenStore[data.token]
         log.info(f"[System][Hutao Login Gateway] {data}")
         uid = data.genshin.uid
-        user_id = data.genshin.userid
+        user_id = data.discord.user_id
```

Comment:
Recommend use `discord.user_id` because in `genshin` field. Some player date doesn't send `user_id` in `genshin` field.